src/app/carrer/[id]/page.tsx

```
@@ -0,0 +1,183 @@
+
+          {/* 記事の本文 */}
+          <article className="rounded-xl border border-gray-100 bg-white p-8 shadow-md">
+            <Suspense
+              fallback={
+                <div className="h-96 animate-pulse rounded-md bg-gray-200"></div>
+              }
+            >
+              <div className="blog-content text-gray-800">
+                <div dangerouslySetInnerHTML={{ __html: blog.contents }} />
+              </div>
+            </Suspense>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)